deploy/cluster_role.yaml

```diff
@@ -1163,3 +1163,19 @@ rules:
   - '*'
   verbs:
   - '*'
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: hco.kubevirt.io:config-reader
+  namespace: kubevirt-hyperconverged
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  resourceNames:
+  - kubevirt-storage-class-defaults
+  verbs:
+  - get
+  - watch
```

Comment:
Should probably add list here as well.  Watchers typically list all items to fill a cache, then watch

Reply:
I also wouldn't be surprised if this file is generated.

Reply:
Yup, here:  https://github.com/kubevirt/hyperconverged-cluster-operator/blob/master/tools/manifest-templator/manifest-templator.go#L72-L74